src/harvest.cpp

```diff
@@ -44,7 +45,7 @@ const harvest_id &harvest_list::id() const
 
 std::string harvest_list::message() const
 {
-    return message_;
+    return SNIPPET.expand( message_ );
```

Comment:
`message_` is an untranslated string, which is translated after returned from `message()`. So after expanded, the message will no longer be correctly translated. I'd suggest migrating `message_` to `translation` to fix this.

Reply:
Translation for the message string should be handled. Has the system changed since #23057?

Reply:
Yes, we now have the `translation` class, though that's not why this.change would cause translation issues. `message_` is read and stored as a raw string, and now gets expanded before being translated, which means the subsequent translation call will no longer have effect.